index.html

```diff
@@ -109,10 +109,10 @@ <h2 class="section-heading">Computing Club Events</h2>
                 </div>
                 <div class="col-md-4 portfolio-item">
 					<a href="#posterModal-3" class="portfolio-link" data-toggle="modal">
-						<img src="img/portfolio/superstar.jpg" class="img-responsive pi-border" alt="">
+						<img src="img/portfolio/alumni.jpg" class="img-responsive pi-border" alt="">
```

Comment:
You should also update the link in "Poster Modal Section". This poster would correspond to Poster Modal 1. I have already updated it for you, but next time pls take note as well!